src/pages/en/guides/migration-guide/from-gatsby.md

```diff
@@ -0,0 +1,1199 @@
+---
+title: Migrating from Gatsby
+description: Tips for migrating an existing Gatsby project to Astro
+layout: ~/layouts/CMSLayout.astro
+stub: false
+framework: Gatsby
+---
+
+Here are some tips for converting a Gatsby project to Astro. This is not a full, step-by-step walkthrough, but it will guide you through some changes you will have to make. 
+
+## Key Similarities
+
+While we'll touch on the differences between Gatsby and Astro shortly, it's important to note the ways in which the two overlap.
+
+- The syntax of `.astro` files is similar to JSX. Writing Astro should feel familiar.
+
+- Astro is component-based. Your markup structure will likely align closely before and after your migration.
+
+- Astro has built-in support for Markdown and an integration for using MDX files. Both Gatsby and Astro use [Remark](https://remark.js.org/) by default for Markdown manipulation and pre-processing, so you can continue to use many of your existing Remark plugins.
+
+- Astro also has [official integrations for using React components](/en/guides/integrations-guide/react/). Note that in Astro, React files **must** have a `.jsx` or `.tsx` extension.
+
+- Astro projects can also be SSG or SSR. (Support for per-page rendering strategy is planned.)
+
+- Astro has support for NPM package usage, including several for React. You may be able to keep some or all of your existing React components and dependencies.
+
+## Key Differences
+
+Now that you understand the ways that Astro and Gatsby align, how do they diverge?
```

Comment:
The following suggestion was originally written by @sarah11918:

```suggestion
When you rebuild your Gatsby site in Astro, you will notice some important differences, especially in how your project data is queried and used to create your site.
```

rebuild? recreate? port? migrate? convert? I'm torn between intentionally using a new word that might bring us new hits, vs a word that maybe feels a bit more natural/appropriate in context? Thoughts?

Also, again here I'm not entirely convinced that any sentence is necessary (is having a preamble just a wordier way to say, "differences!") BUT if we do use it, I'm thinking maybe to make it useful, a nod to an actual key difference like the GraphQL data layer, and `createPates`? Again, open to thoughts here!

(Continued point: leaning away from self-narration of the page)

Reply:
I think any of the words you suggested here feel pretty appropriate. In addition to bringing us new potential SEO, it also helps break up the "monotony" of reading in-depth content to have our terminology shift a _little_ to mix things up.

As for pointing towards a key difference, I'm not sure I'm entirely sold simply because I feel like people might glance over this sentence and assume it's the _primary_ difference between Astro and Gatsby, despite there being a couple (which is the point of this section)